### PR TITLE
docs: updating readme

### PR DESCRIPTION
**Pull request by khusharth**
Hey! I have added something new!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,5 @@
-# comp-lib-demo
\ No newline at end of file
+# comp-lib-demo
+
+ksjcjsnvkjn
```
Comment (khusharth) [inline README.md:3]: Hey! This can be done like this: kwscjsdkjds

File: README.md [hunk 2]
```diff
@@ -1 +1,5 @@
...
+sdvknjvnkjdfn
+ksfdlkfvjnfkjnvjfdn
```
Comment (khusharth) [inline README.md:5]: this is not the best approach.

**Review by khusharth — COMMENTED** [review]
Hey! @khusharth I have reviewed your code. 
Some changes related to UI:
- efdf
- svfd

File: README.md (hunk 1 above)
Comment (khusharth) [reply README.md:3]: hey, i have solved this now.

File: README.md [hunk 3]
```diff
@@ -1 +1,5 @@
-# comp-lib-demo
\ No newline at end of file
+# comp-lib-demo
+
+ksjcjsnvkjn
+sdvknjvnkjdfn
```
Comment (asyriac) [inline README.md:4]: Test comment for review.

**Comment by Jaynil1611** [comment]
This section can be converted into a good discussion!

File: README.md (hunk 3 above)
Comment (sjsouvik) [inline README.md:4]: Nice demo Khusharth!

**Review by vinaybajjuri58 — COMMENTED** [review]
Added demo text

File: README.md (hunk 2 above)
Comment (aneenajohn) [inline README.md:5]: Hey! I can review this.

**Comment by vikasvk11** [comment]
Great approach!

File: README.md (hunk 1 above)
Comment (Jaynil1611) [inline README.md:3]: This is not English

**Review by Jaynil1611 — COMMENTED** [review]
Fix typos

File: README.md (hunk 3 above)
Comment (sjsouvik) [reply README.md:4]: we can try it in our projects

File: README.md (hunk 1 above)
Comment (vikasvk11) [inline README.md:3]: I think this can be considered as review

**Comment by yogeshpendse** [comment]
This is a demo comment

**Review by vikasvk11 — COMMENTED** [review]
Submitting a general feedback without explicit approval.

File: README.md (hunk 2 above)
Comment (khusharth) [reply README.md:5]: Hey! I have fixed this now.

Comment (aneenajohn) [reply README.md:5]: Good Job!

File: README.md [hunk 4]
```diff
@@ -1 +1,5 @@
-# comp-lib-demo
\ No newline at end of file
+# comp-lib-demo
```
Comment (khusharth) [inline README.md:1]: hey

**Review by khusharth — COMMENTED** [review]
Hey! I am done with reviewing your code.

**Pull request merged**